template/next-ts-chakra-ui/sentry.server.config.js

```diff
@@ -0,0 +1,14 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || '';
+const SENTRY_ENVIRONMENT = process.env.SENTRY_ENVIRONMENT || '';
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  enabled: SENTRY_ENVIRONMENT !== '',
+  tracesSampleRate: 0.01,
```

Comment:
Still not sure I fully understand how this gets used - would it make any sense to have the value in ENV so it could eg be different in staging/uat v production?